e2e.Dockerfile

```diff
@@ -6,12 +6,13 @@ WORKDIR /src/app/
 COPY go.mod go.sum* ./
 RUN go mod download
 COPY . .
+ENV KO_DOCKER_REPO=ghcr.io/cosmos
```

Comment:
Think we should explicitly specify gaia repo, otherwise will create something else
export KO_DOCKER_REPO=ghcr.io/cosmos/gaia

Reply:
Hi, I m not sure about this, because you see here does not have a repo name.https://github.com/orgs/cosmos/packages